agent/engine/dockerstate/docker_task_engine_state.go

```diff
@@ -271,6 +277,25 @@ func (state *DockerTaskEngineState) ContainerMapByArn(arn string) (map[string]*a
 	return ret, ok
 }
 
+// PulledContainerMapByArn returns a map of pulled containers belonging to a particular task ARN
+func (state *DockerTaskEngineState) PulledContainerMapByArn(arn string) (map[string]*apicontainer.DockerContainer, bool) {
+	state.lock.RLock()
+	defer state.lock.RUnlock()
+
+	ret, ok := state.taskToPulledContainer[arn]
```

Comment:
nit: where possible, it's better (easier to read) to align to the left, so this code could be expressed as 

```golang
	ret, ok := state.taskToPulledContainer[arn]
	if !ok {
		return ret, ok
	}
  	mc := make(map[string]*apicontainer.DockerContainer)
  	for k, v := range ret {
		mc[k] = v
	}
        return mc, ok
```

Reply:
`PulledContainerMapByArn` and `ContainerMapByArn` are updated with this suggestion. Thank you!